nuxt.config.ts

```diff
@@ -0,0 +1,60 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+  devtools: { enabled: true },
+  app: {
+    head: {
+      link: [{ rel: 'icon', type: 'image/png', href: '/images/favicon.png' }],
+      title: 'LUKSO Wallet',
+    },
+  },
+  modules: [
+    '@nuxtjs/tailwindcss',
+    [
+      '@pinia/nuxt',
+      {
+        autoImports: ['defineStore', 'acceptHMRUpdate'],
+      },
+    ],
+    '@nuxtjs/device',
+    '@nuxtjs/plausible',
+  ],
+  plausible: {
+    domain: 'migrate.lukso.network',
+  },
```

Comment:
This is not relevant anymore. We can remove it.

Reply:
Ah right domain is wrong, we not planning to use plausible for wallet?

Reply:
We can, ok lets add it!